addons/editor-theme3/addon.json

```diff
@@ -104,6 +104,16 @@
       "default": "#0FBD8C",
       "allowTransparency": true
     },
+    {
+      "name": "Scratch Addons blocks",
```

Comment:
Please change this

Reply:
To what? And why?

Reply:
To "Debugger" because that's currently the only addon that adds new blocks. I'm not sure which name is better.

Reply:
There's an open PR that I'll probably modify so that it's part of a separate addon instead of debugger (comment blocks)
So I don't think this should be changed

Reply:
I would personally prefer anything that doesn't mention "Scratch Addons" by name but will change it myself for TW if that doesn't happen

Reply:
"Addon blocks" is another option, but I think "Scratch Addons blocks" is clearer.